packages/core/src/hooks/useAddressBook.tsx

```diff
@@ -1,29 +1,36 @@
 import { useState, useEffect } from 'react';
 
 export default function useAddressBook(): [
-  IAddressContact[] | undefined, // contacts
+  AddressContact[] | undefined, // contacts
   (
     name: string,
-    addresses: IContactAddress[],
-    dids: IContactDID[],
+    addresses: ContactAddress[],
+    dids: ContactDID[],
     notes: string,
     nftid: string,
-    domainnames: IContactDomainName[]
+    domainNames: ContactDomainName[]
   ) => void, // addContact
-  (contactid: number) => void, // removeContact
-  (contactid: number) => IAddressContact | undefined, // getContactContactId
-  (contact: IAddressContact, contactid: number) => void,
-  (address: string) => IAddressContact | undefined // getContactByAddress
+  (contactId: number) => void, // removeContact
+  (contactId: number) => AddressContact | undefined, // getContactContactId
+  (
+    name: string,
+    addresses: ContactAddress[],
+    dids: ContactDID[],
+    notes: string,
+    nftid: string,
+    domainNames: ContactDomainName[]
+  ) => void, // editContact
+  (address: string) => AddressContact | undefined // getContactByAddress
 ] {
   // editContact
-  const [addressBook, setAddressBook] = useState<IAddressContact[]>([]);
+  const [addressBook, setAddressBook] = useState<AddressContact[]>([]);
   const LOCAL_STORAGE_KEY = 'addressbook';
   const contactBookJSON = JSON.stringify(addressBook);
   useEffect(() => {
```

Comment:
Both of these useEffect hooks can be removed since we should be using `useLocalStorage`